arbeitszeit_web/www/presenters/request_user_password_reset_presenter.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases import request_user_password_reset as use_case
+
+class RequestUserPasswordResetViewModel:
+    # @TODO: Does this need to be implemented?
```

Comment:
Is this class supposed to create a page that confirms that the user has successfully changed his or her password?  If the answer to your TODO question is "no," how does the user get confirmation that the password change has succeeded?